common/src/main/java/org/dash/wallet/common/services/analytics/AnalyticsService.kt

```diff
@@ -31,6 +32,10 @@ class FirebaseAnalyticsServiceImpl: AnalyticsService {
     private val crashlytics = Firebase.crashlytics
 
     override fun logEvent(event: String, params: Bundle) {
+        if (BuildConfig.DEBUG) {
+            return
+        }
+
```

Comment:
This is an interesting idea as it will exclude developer actions when running and testing debug builds from the events in firebase.